src/guiguts/maintext.py

```diff
@@ -2829,10 +2834,28 @@ def highlight_cursor_line(self) -> None:
             row = self.get_insert_index().row
             self.tag_add(HighlightTag.CURSOR_LINE, f"{row}.0", f"{row+1}.0")
 
-    def highlight_initialize_tags(self) -> None:
-        """Initialize highlighting tags and determine their prioritization.
-        Intended to run once at startup.
+    def highlight_configure_tags(self, **kwargs: Any) -> None:
```

Comment:
Is this equivalent to
```
def highlight_configure_tags(self, first_run: bool = False) -> None:
```
then later instead of 
```
if "first_run" in kwargs and kwargs["first_run"]:
```
using
```
if first_run:
```
If so, then I think the latter would be clearer, but I may have missed some subtle difference.

Reply:
Functionally those are equivalent, yes. I actually started with that form. I changed it because the initial call had a "magic value" argument that I thought read less clearly. That is, using a positional argument, your call in `__init__` would be:

```python
self.highlight_configure_tags(True)
```

Which works, but it's not clear what True means without checking the function definition.

Reply:
I can see your point about being less readable on the other end (in the function definition), but I was hoping the docstring would clarify it enough.

Reply:
On nomenclature, I thought a keyword argument was just one that had a default value in the function def, so I think in my suggestion `first_run` is still a keyword argument. The `kwargs` thing is when you have some "unknown" keyword args.

Reply:
Just looking up some Python stuff to try to understand the nomenclature better

Reply:
*smacks self in forehead*

Well, I learned something new today! I didn't realize you could pass the name of a positional argument! I wonder if that changed in the Python 2 to 3 shift... I was 99% sure that didn't work...

Anyway, it does work, and I just pushed a new version with that change.

Reply:
NTS: This answer was the most helpful: https://stackoverflow.com/a/57819001